ui/jobs/components/drk.ts

```diff
@@ -33,7 +35,7 @@ export function setup(bars) {
       p.classList.remove('mid');
     }
 
-    const oldSeconds = parseFloat(darksideBox.duration) - parseFloat(darksideBox.elapsed);
+    const oldSeconds = darksideBox.duration - darksideBox.elapsed;
```

Comment:
How does this work? Aren't these actually strings? I'm looking at `timerbox.ts` which says `get duration(): string | null`.